### ISSUE TITLE
[Bug] The login feature of the deployed application doesn't work

### ISSUE THREAD

**Issue by longnd** [dungtruongtien/kewe-crawler#19]
#Issue
----
The [deployed](https://github.com/dungtruongtien/kewe-crawler#uat-environment) application, found at https://kewecrawling.info/ failed to let the user login. I tried with the provided account (admin@gmail.com/abc123) as well as registering a new account, both failed with a general error:
| Logging in | Registering |
| -- | -- |
| <img width="1426" alt="image" src="https://github.com/dungtruongtien/kewe-crawler/assets/1896814/db1995a4-7725-4f54-99f7-51aaf204a6f4"> | <img width="1301" alt="image" src="https://github.com/dungtruongtien/kewe-crawler/assets/1896814/489068d4-e7a3-4bf5-ba8f-9c21406302b3">|

The same issue happens if running the application locally using Docker

#Expected
--
The login & registration feature should work as expected.

#Steps to reproduce
1. Go to https://kewecrawling.info/ 
2. Login using this user (admin@gmail.com/abc123) or registering for a new account
3. Both features don't work

**Comment by dungtruongtien** [comment]
Sorry about this inconvenience. I tested all of this before submitting to you (both locally and deployed environment) and it all worked, I'll trace and try to fix it in deployed environment later. 

Please help me to check that when you run locally. Is there any problem with any containers? Did they all start well? Did `init.sql` was execcuted in postgres when initiation?

**Comment by longnd** [comment]
> Did they all start well

Yes, all containers run without any issues; the web app functioned and sent the request to the API; as you can see in the screenshot in the first comment, it's the API endpoint failed (INTERNAL_ERROR)

I did not try to investigate further, and it took time, I ended up setting up the services directly on my local (without Docker) instead and managed to have all of the apps up and running
